sdk/ml/azure-ai-ml/azure/ai/ml/operations/_workspace_operations_base.py

```diff
@@ -114,6 +115,26 @@ def begin_create(
 
         workspace.resource_group = resource_group
         template, param, resources_being_deployed = self._populate_arm_paramaters(workspace, **kwargs)
+        # check if create with workspace hub request is valid
+        if workspace._kind == PROJECT_WORKSPACE_KIND and workspace.workspace_hub:
+            if not all(
+                x is None
+                for x in [
+                    workspace.storage_account,
+                    workspace.container_registry,
+                    workspace.key_vault,
+                    workspace.public_network_access,
+                    workspace.managed_network,
+                    workspace.customer_managed_key,
+                ]
+            ):
```

Comment:
Do all of these have to be None to create a project workspace? I thought PNA can be set for project workspaces.

Reply:
According to our code, we override the PNA flag with workspaceHub's PNA during creation, so setting a PNA flag here is  confusing.